app/models/story.rb

```diff
@@ -1,41 +1,22 @@
 class Story < ApplicationRecord
+  MEDIA_PATH = Rails.env.test? ? 'spec/fixtures/media' : 'media'
```

Comment:
The media path is used by other models as well. I was thinking, why not add it as a custom Rails configuration variable, and set it in the different environment configs. Like:
```
# application.rb
Rails.application.configure do
  config.media_path = 'media'
end
```
overwrite it in `test.rb` and then just call:
```
Rails.configuration.media_path
```

Reply:
@namel3ss Can be done. I thought of doing this initially, but then thought to keep it the was it is currently until @MxOliver or @kalimar takes a pass at reviewing this PR. Also, thank you for waiting on this PR for your #300

Reply:
I think that sounds like a good idea, @kalimar thoughts? Also I've approved this PR so you can go ahead and merge so that @namel3ss can rebase